### PR TITLE
Polish language pack

### PR DESCRIPTION
**Pull request by Epacik**
I want to use HandyControls in my future projects and I don't want to create a language pack for each of them (If I won't other users with a system language set to Polish would see Chinese characters in e.g. message boxes), so I decided to add a Polish language pack to the HC itself.

I tried to create pull request [here](https://github.com/ghost1372/HandyControls/pull/46) but I was advised to create it in your repository instead.

### CODE REVIEW

**Comment by NaBian** [comment]
great! we will merge this pr if you can add pack for hc demo

**Comment by Epacik** [comment]
It's already added

**Comment by NaBian** [comment]
tnx, just wait for some days (not more than 3 days), cos we have other high priority tasks.

File: src/Shared/HandyControlDemo_Shared/Properties/Langs/Lang.pl.resx [hunk 1]
```diff
@@ -358,7 +355,7 @@
     <value>Czy uruchomić ponownie aby zmienić język?</value>
   </data>
   <data name="About" xml:space="preserve">
-    <value>O...</value>
+    <value>Około</value>
```
Comment (Epacik) [inline src/Shared/HandyControlDemo_Shared/Properties/Langs/Lang.pl.resx:358]: "Około" in Polish is most often used as "around" and not "about".
Most programs and companies use "O something" when they mean "About something".
Example from Firefox:
obraz
obraz

Alternatively "about" alone could be translated as "Informacje" ("Informations"), as translated in VS Code.
obraz
obraz

Comment (NaBian) [reply src/Shared/HandyControlDemo_Shared/Properties/Langs/Lang.pl.resx:358]: should it be O... or O?

Comment (Epacik) [reply src/Shared/HandyControlDemo_Shared/Properties/Langs/Lang.pl.resx:358]: After a longer consideration, I came to a conclusion, that "Informacje" would look better in there.
Both "O" and "O..." would be a little too short

Comment (NaBian) [reply src/Shared/HandyControlDemo_Shared/Properties/Langs/Lang.pl.resx:358]: update in 1ae275a2

**Pull request merged**